### PR TITLE
Implement enabling/disabling of specific notifications

### PR DESCRIPTION
**Pull request by mmilosevic10**

### CODE REVIEW

File: pushkin/database/database.py [hunk 1]
```diff
@@ -184,6 +184,9 @@ def process_user_login(login_id, language_id, platform_id, device_id, device_tok
     session.commit()
     session.close()
 
+def process_turn_off_notification():
```
Comment (dprokic) [inline pushkin/database/database.py]: This function is empty? Is it needed?

File: pushkin/database/migrations/versions/61e52d26ebea_message_blacklist.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+"""message_blacklist
```
Comment (dprokic) [inline pushkin/database/migrations/versions/61e52d26ebea_message_blacklist.py]: Replace this comment with License.

File: pushkin/database/database.py (hunk 1 above)
Comment (mmilosevic10) [inline pushkin/database/database.py]: I will remove it :)

File: pushkin/database/migrations/versions/61e52d26ebea_message_blacklist.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    context = op.get_context()
+    connection = op.get_bind()
+
+    if not context.dialect.has_table(connection.engine, 'message_blacklist'):
```
Comment (dprokic) [inline pushkin/database/migrations/versions/61e52d26ebea_message_blacklist.py]: Remove this if. Table should not exist, because it is not created in previous revisions. If it exist by chance, that is a problem and upgrade should fail.

File: pushkin/tests/test_server_proto.py [hunk 4]
```diff
@@ -154,6 +156,20 @@ def test_post_event(setup_database, mock_processor, http_client, post_event_url,
...
+    '''Test that a valid request is succesfully parsed in post_event'''
+    context.request_processor.submit.return_value = True
+
+    login = database.upsert_login(123, 7)
+    database.upsert_message_blacklist(123, [1])
+    context.request_processor.submit.return_value = True
+    request = tornado.httpclient.HTTPRequest(post_event_url, method='POST', body=event_batch_proto)
+    response = yield http_client.fetch(request)
+    assert not context.request_processor.sender_manager.submit.called
+    database.upsert_message_blacklist(123, [100])
```
Comment (dprokic) [inline pushkin/tests/test_server_proto.py]: Why did you change blacklist at the end? Did you want to test something more?

Comment (mmilosevic10) [inline pushkin/tests/test_server_proto.py]: Obsolete line. WIll remove
